samples/snapshot/pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-bigquery</artifactId>
-      <version>1.110.1</version>
+      <version>1.110.2-SNAPSHOT</version>
```

Comment:
This should be handled by release-please. Can you fix the version replace tag on line 42?

Reply:
okay sure i will move it to line 43